packages/app/src/pages/actueel/gemeente/[code].tsx

```diff
@@ -359,7 +359,13 @@ const TopicalMunicipality = (props: StaticProps<typeof getStaticProps>) => {
               </ChoroplethTwoColumnLayout>
 
               <Box mt={4}>
-                <EscalationLevelExplanations />
+                <EscalationLevelExplanations
+                  hasUnknown={
+                    choropleth.vr.escalation_levels.filter(
+                      (item) => item.level === null
+                    ).length > 0
```

Comment:
```suggestion
                    some(choropleth.vr.escalation_levels, x => !isPresent(x))
```

I think this should work using lodash some and ts-is-present